src/app/recent_files.h

```diff
@@ -22,13 +23,18 @@ namespace app {
            kPinnedFolders,
            kRecentFolders,
            kCollections };
+
   public:
+    static constexpr std::array<const char*, 4> sections{
+      "PinnedFiles", "RecentFiles", "PinnedPaths", "RecentPaths"
+    };
+
```

Comment:
I moved it to make it publicly available and easily iterable in a for range, to avoid having the list of sections in multiple places.

Reply:
Probably I missed something in this commit, but it looks like the `sections` array is not used outside `src/app/recent_files.cpp`, so we should avoid a change that is not required in the PR. I think we can safely revert this change.

Even if this `sections` constant needs to be public for this PR, we prefer a `k` prefix and `UpperCase` kind of name (like `kSectionName`) for constants (instead of `sections` which can be confused with local variable names).

Reply:
Oh, my bad, it's a leftover from something I thought I was still using, I've just reverted it.